Lib/fontParts/base/lib.py

```diff
@@ -337,14 +343,14 @@ def get(self, key: str, default: Optional[LibValueType] = None) -> LibValueType:
         """
         return super(BaseLib, self).get(key, default)
 
-    def items(self) -> List[Tuple[str, LibValueType]]:
-        """Return an unordered list of the lib's items.
+    def items(self) -> BaseItems[Tuple[str, LibValueType]]:
+        """View the lib's items.
```

Comment:
Maybe same here `Returns the lib's items`?

Reply:
Fixed!